docs-ref-services/latest/private-dns.md

```diff
@@ -0,0 +1,32 @@
+# Azure Private DNS libraries
+
+## Overview
+
+Azure Private DNS provides a reliable and secure DNS service for your virtual network.
+
+To get started with Azure Private DNS, see [Create an Azure private DNS zone using the Azure portal](/azure/dns/private-dns-getstarted-portal).
+
+## Management API
+
+Create, configure and manage and scale private dns in Azure from your code with the management API.
+
+Install the library via pip.
+
+```bash
+pip install azure.mgmt.privatedns
+```
+
+> [!div class="nextstepaction"]
+> [Explore the Management APIs](/python/api/azure-mgmt-privatedns/azure.mgmt.privatedns)
+
```

Comment:
I don't know the effect of line19-20. Is it expected?
![image](https://user-images.githubusercontent.com/70930885/167792164-dd49e9ed-6c70-4846-a0a9-0f95744362f3.png)

Reply:
@msyyc Yes it is required as we need a button to help users navigate to the Management API. 
Sharing a sample screenshot if that helps.

![image](https://user-images.githubusercontent.com/57353862/167793236-b04416a5-0a16-4189-a49a-f5fd5a934bb0.png)